src/gluonnlp/data/transforms.py

```diff
@@ -881,6 +895,14 @@ def _is_whitespace(self, char):
             return True
         return False
 
+    def _whitespace_tokenize(self, text):
+        """Runs basic whitespace cleaning and splitting on a piece of text."""
+        text = text.strip()
+        if not text:
```

Comment:
The if is not needed. If the string is empty, an empty list is returned by split(). Or do we need to handle None?

Reply:
Yes, thank you for your reminder, it has been revised.